jupyter-org-client.el

```diff
@@ -416,6 +416,357 @@ the `syntax-table' will be set to that of the REPL buffers."
   (jupyter-org-with-src-block-client
    (cl-call-next-method)))
 
+;;; Org helper functions
+
+;;;###autoload
+(defun jupyter-org-insert-src-block (&optional below)
+  "Insert a src block above the current point.
+With prefix arg BELOW, insert it below the current point.
+
+If point is in a block, copy the header to the new block"
+  (interactive "P")
+  (if (org-in-src-block-p)
+      (let* ((src (org-element-context))
+             (start (org-element-property :begin src))
+             (end (org-element-property :end src))
+             (lang (org-element-property :language src))
+             (switches (org-element-property :switches src))
+             (parameters (org-element-property :parameters src))
+             location)
+        (if below
+            (progn
+              (goto-char start)
+              (setq location (org-babel-where-is-src-block-result nil nil))
+              (if (not  location)
+                  (goto-char end)
+                (goto-char location)
+                (goto-char (org-element-property :end (org-element-context))))
+              (insert (format "#+BEGIN_SRC %s\n\n#+END_SRC\n\n"
```

Comment:
You can let `org-element-interpret-data` do most of the work here.

```emacs-lisp
(insert
 (org-element-interpret-data
  (org-element-put-property src :value "")))
```

should make all of the lang, switches, and parameters variables unnecessary.

Reply:
I've switched to `org-element-interpret-data`. 
The only remaining questions is if we do anything with the "src_block" vs "SRC_BLOCK".
In my version, I was using uppercase letters. The `org-element-interpret-data` function uses only lower case letters.

Reply:
That is unfortunate, the problem with hard-coding uppercase here is what if someone has all their code blocks in all lowercase which seems to be the default behavior now in `org-mode` 9.2 (https://lists.gnu.org/archive/html/emacs-orgmode/2019-02/msg00148.html).

I guess the simplest solution would be to avoid using `org-element-interpret-data` and have a customizable variable, `jupyter-org-prefer-uppercase-src-blocks` which defaults to 
`(version< (org-version) "9.2")` and use that to determine if uppercase or lowercase block delimiters should be inserted.

Reply:
I was not aware that the new standard was to use lower case for all org keywords.
As long as it's consistent, I don't mind then the lower case keywords.
I wouldn't mind then using `org-element-interpret-data` and letting org-mode choose the case.

What would you prefer?